tests/test_settings.py

```diff
@@ -90,3 +99,105 @@ class TestClass:
+    from krux.input import BUTTON_ENTER, BUTTON_PAGE, BUTTON_PAGE_PREV
+    from krux.krux_settings import Settings, NumberSetting
+
+    cases = [
+        (  # 0
+            (
+                BUTTON_PAGE,
+                # Encryption
+                BUTTON_ENTER,
+                # pbkdf2
+                BUTTON_ENTER,
+                # go to "<" keypad position
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                # delete last 0
+                BUTTON_ENTER,
+                # go to "1" keypad
+                BUTTON_PAGE,
+                BUTTON_PAGE,
+                BUTTON_PAGE,
+                BUTTON_PAGE,
+                # Enter "1"
+                BUTTON_ENTER,
+                # go to "Go" keypad position
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                # Go
+                BUTTON_ENTER,
+                # Leave Encryption settings
+                BUTTON_PAGE_PREV,
+                BUTTON_ENTER,
+                # Leave Settings
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                BUTTON_ENTER,
+            ),
+            # Assert iterations didn't change to 100,001 (not multiple of 10,000)
+            lambda: Settings().encryption.pbkdf2_iterations == 100000,
+            NumberSetting,
+        ),
+        (  # 0
+            (
+                BUTTON_PAGE,
+                # Encryption
+                BUTTON_ENTER,
+                # pbkdf2
+                BUTTON_ENTER,
+                # go to "<" keypad position
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                # delete last 5 zeros
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                # go to "1" keypad
+                BUTTON_PAGE,
+                BUTTON_PAGE,
+                BUTTON_PAGE,
+                BUTTON_PAGE,
+                # Enter "1"
+                BUTTON_ENTER,
+                # Go to Zero
+                BUTTON_PAGE_PREV,
+                # Enter 4 zeros
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                BUTTON_ENTER,
+                # go to "Go" keypad position
+                BUTTON_PAGE_PREV,
+                # Go
+                BUTTON_ENTER,
+                # Leave Encryption settings
+                BUTTON_PAGE_PREV,
+                BUTTON_ENTER,
+                # Leave Settings
+                BUTTON_PAGE_PREV,
+                BUTTON_PAGE_PREV,
+                BUTTON_ENTER,
+            ),
+            # Assert iterations changed to a multiple of 10,000
+            lambda: Settings().encryption.pbkdf2_iterations == 110000,
+            NumberSetting,
+        ),
+    ]
+    for case in cases:
+        ctx = create_ctx(mocker, case[0])
```

Comment:
Context and Login objects are not necessary, you could call the Settings functions as other examples on this file. So all the navigation procedures would also be removed from the test